README.md

```diff
@@ -188,10 +188,8 @@ const Component = React.lazy(() => {
   switch (effectiveType) {
     case "3g":
       return import(/* webpackChunkName: "light" */ "./light.js");
-      break;
```

Comment:
Changes above LGTM but I would like for us to maintain the break statements here. Could we bring these back?

Reply:
@addyosmani thanks for the feedback! I've reverted the changes :)